src/Control/Monad/Aff/AVar.js

```diff
@@ -81,8 +81,8 @@ exports._killVar = function (nonCanceler, avar, e) {
 
         try {
           consumer.error(e);
```

Comment:
There's a reference to `e` here, is this correct or should this have been updated?